relentless/potential/potential.py

```diff
@@ -363,27 +447,67 @@ def derivative(self, pair, param, r):
         return deriv
 
     def _zeros(self, r):
-        # coerce input shape and create zeros for output
+        """Coerces input shape and creates zeros for output.
```

Comment:
This was a silly description by me, let's make it:
"Force input to a one-dimensional array and make matching array of zeros."